sirepo/uri_router.py

```diff
@@ -116,6 +118,11 @@ def init(app, simulation_db):
 
     for n in _REQUIRED_MODULES + tuple(sorted(feature_config.cfg().api_modules)):
         register_api_module(importlib.import_module('sirepo.' + n))
+    for f in pkio.sorted_glob(simulation_db.STATIC_FOLDER.join('en','*.html')):
```

Comment:
We don't necessarily want these all at the top level. 

This is in an awkward place in the code. It's very detailed code when the rest is fairly highly-level code. I would embed in a method or put in init_uris.